src/components/focus_trap/focus_trap.spec.tsx

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { mount } from '@cypress/react';
+import { EuiFocusTrap } from './focus_trap';
+import { EuiPortal } from '../portal';
+
+function add(a: number, b: number) {
+  return a + b;
+}
+add('3', 5);
+
+describe('EuiFocusTrap', () => {
+  describe('behavior', () => {
+    describe('focus', () => {
```

Comment:
If you're not examining any behavior other than focus, I'd suggest just making this a single describe block:
```suggestion
  describe('focus behavior', () => {
```

Reply:
I've removed the wrapping `behavior` describe and left its 2 children describes. The `behavior` level is a potentially interesting separation in the jest tests, but isn't here.